source/chapters/appendix.rst

```diff
@@ -165,9 +159,9 @@ language settings the defaults may be different.
 +----------------------------------------+-----------------------------------------+------------------------------------------+----------------------------+
 | Start/Stop                             |                                         |                                          | :kbd:`Return`              |
 +----------------------------------------+-----------------------------------------+------------------------------------------+----------------------------+
-| Seek Backwards                         |                                         |                                          | :kbd:`Alt` + :kbd:`Left`   |
+| Seek Backwards                         |                                         |                                          | :kbd:`Alt` + :kbd:`←`      |
 +----------------------------------------+-----------------------------------------+------------------------------------------+----------------------------+
-| Seek Forwards                          |                                         |                                          | :kbd:`Alt` + :kbd:`Right`  |
+| Seek Forwards                          |                                         |                                          | :kbd:`Alt` + :kbd:`→`      |
```

Comment:
Please revert that since we agreed to keep the literal key names because there are obviously no icons that work reliably on all platforms.

Reply:
I thought we just said we use the regular arrows instead of the more obscure ones.